docs/src/quickstart.md

```diff
@@ -2,271 +2,243 @@
 
-basis = Basis(observables, u)
+```@example 1
+system = result(res);
+params = parameters(res);
+println(system) #hide
+println(params) #hide
 ```
 
-A `Basis` captures a bunch of functions defined over some variables provided via [ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl).
-Here, we included the state and `u[1]^2`. Now, we simply call `gEDMD`, which
-will compute the generator of the [Koopman Operator](@ref koopman_operator) associated with the model.
+which is indeed our pendulum model with a slight offset due to the noisy measurements and the estimation of the time derivates.
 
-```@example 2
-approximation = gEDMD(X, DX, basis)
+Since any system obtained via a `solve` command is a [`Basis`](@ref) and hence a subtype of an `AbstractSystem` defined in `ModelingToolkit`, we can simply simulate the result via:
+
+```@example 1
+infered_prob = ODEProblem(system, u0, tspan, parameters(res))
+infered_solution = solve(infered_prob, Tsit5(), saveat = ts)
+plot(infered_solution, label = ["Infered" nothing], color = :red) # hide
 
-approximation_problem = ODEProblem(approximation, u0, tspan)
-generator_sol = solve(approximation_problem, Tsit5(), saveat = solution.t)
+function pendulum(u, p, t) # hide
+    x = u[2] # hide
+    y = -9.81sin(u[1]) - 0.3u[2]^3 -3.0*cos(u[1]) # hide
+    return [x;y] # hide
+end # hide
 
-plot(generator_sol, label = ["u[1]" "u[2]"]) #hide
-scatter!(solution, label = ["True u[1]" "True u[2]"]) #hide
-savefig("slow_approximation_cont.png") #hide
-scatter(eigvals(approximation), label = "Estimate") # hide
-scatter!(eigvals([p[1] 0 0; 0 p[2] -p[2]; 0 0 2*p[1]]), label = "True", legend = :bottomright) #hide
-savefig("eigenvalue_slowmanifold.png") #hide
+prob = ODEProblem(pendulum, u0, tspan) # hide
+sol = solve(prob, Tsit5(), saveat = 0.01) # hide
 
+plot!(sol, label = ["Ground Truth" nothing], color = :black, style = :dash) # hide
+savefig("SINDy_Example_Data_Infered.png") #hide
 ```
-![](slow_approximation_cont.png)
 
-Looking at the eigenvalues of the system, we see that the estimated eigenvalues of the linear system are close to the true values.
+!!! warning
+    As of now, the control input is dropped in the simulation of a system. We are working on this and pull requests are welcome!
 
-![](eigenvalue_slowmanifold.png)
+![](SINDy_Example_Data_Infered.png)
 
-## Nonlinear Systems - Sparse Identification of Nonlinear Dynamics
+As we can see above, the estimated system matches the ground truth reasonably well.
 
-Okay, so far we can fit linear models via DMD and nonlinear models via EDMD. But what if we want to find a model of a nonlinear system *without moving to Koopman space*? Simple, we use [Sparse Identification of Nonlinear Dynamics](https://www.pnas.org/content/113/15/3932) or `SINDy`.
 
-As the name suggests, `SINDy` finds the sparsest basis of functions which build the observed trajectory. Again, we will start with a nonlinear system
+## Implicit Nonlinear Dynamics
```

Comment:
This section returns the correct equations, but I am unable to transform the Basis via structural simplification / alias_elimination etc.

Reply:
I will take a look tonight. Please, ping me again if I haven't replied by tomorrow.

Reply:
That's stange. Inside a new environment, everything runs smoothly for me:

```bash
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.1 (2021-04-23)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.6) pkg> activate .
  Activating new environment at `~/Documents/code/test_data/Project.toml`

(test_data) pkg> st
      Status `~/Documents/code/test_data/Project.toml` (empty project)

(test_data) pkg> add DataDrivenDiffEq#main
    Updating git-repo `https://github.com/SciML/DataDrivenDiffEq.jl.git`
    Updating registry at `~/.julia/registries/General`
   Resolving package versions...
   Installed Compat ─ v3.30.0
    Updating `~/Documents/code/test_data/Project.toml`
  [2445eb08] + DataDrivenDiffEq v0.6.0 `https://github.com/SciML/DataDrivenDiffEq.jl.git#main`
    Updating `~/Documents/code/test_data/Manifest.toml`
  [c3fe647b] + AbstractAlgebra v0.16.0
  [621f4979] + AbstractFFTs v1.0.1
  [1520ce14] + AbstractTrees v0.3.4
  [79e6a3ab] + Adapt v3.3.0
  [ec485272] + ArnoldiMethod v0.1.0
  [4fba245c] + ArrayInterface v3.1.14
  [d360d2e6] + ChainRulesCore v0.9.44
  [b630d9fa] + CheapThreads v0.2.4
  [861a8166] + Combinatorics v1.0.2
  [38540f10] + CommonSolve v0.2.0
  [bbf7d656] + CommonSubexpressions v0.3.0
  [34da2185] + Compat v3.30.0
  [187b0558] + ConstructionBase v1.2.1
  [717857b8] + DSP v0.6.10
  [9a962f9c] + DataAPI v1.6.0
  [2445eb08] + DataDrivenDiffEq v0.6.0 `https://github.com/SciML/DataDrivenDiffEq.jl.git#main`
  [82cc6244] + DataInterpolations v3.3.1
  [864edb3b] + DataStructures v0.18.9
  [e2d170a0] + DataValueInterfaces v1.0.0
  [2b5f629d] + DiffEqBase v6.61.0
  [c894b116] + DiffEqJump v6.14.1
  [163ba53b] + DiffResults v1.0.3
  [b552c78f] + DiffRules v1.0.2
  [31c24e10] + Distributions v0.24.18
  [ffbed154] + DocStringExtensions v0.8.4
  [e2ba6199] + ExprTools v0.1.3
  [8f5d6c58] + EzXML v1.1.0
  [7a1cc6ca] + FFTW v1.4.1
  [7034ab61] + FastBroadcast v0.1.7
  [1a297f60] + FillArrays v0.11.7
  [6a86dc24] + FiniteDiff v2.8.0
  [59287772] + Formatting v0.4.2
  [f6369f11] + ForwardDiff v0.10.18
  [069b7b12] + FunctionWrappers v1.1.2
  [0e44f5e4] + Hwloc v2.0.0
  [615f187c] + IfElse v0.1.0
  [d25df0c9] + Inflate v0.1.2
  [d8418881] + Intervals v1.5.0
  [c8e1da08] + IterTools v1.3.0
  [42fd0dbc] + IterativeSolvers v0.9.1
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [692b3bcd] + JLLWrappers v1.3.0
  [b964fa9f] + LaTeXStrings v1.2.1
  [2ee39098] + LabelledArrays v1.6.1
  [23fbe1c1] + Latexify v0.15.5
  [093fc24a] + LightGraphs v1.3.5
  [d3d80556] + LineSearches v7.1.1
  [2ab3a3ac] + LogExpFunctions v0.2.4
  [bdcacae8] + LoopVectorization v0.12.21
  [1914dd2f] + MacroTools v0.5.6
  [e1d29d7a] + Missings v1.0.0
  [78c3b35d] + Mocking v0.7.1
  [961ee093] + ModelingToolkit v5.16.0
  [46d2c3a1] + MuladdMacro v0.2.2
  [d41bc354] + NLSolversBase v7.8.0
  [77ba4419] + NaNMath v0.3.5
  [8913a72c] + NonlinearSolve v0.3.8
  [6fe1bfb0] + OffsetArrays v1.8.0
  [429524aa] + Optim v1.3.0
  [bac558e1] + OrderedCollections v1.4.1
  [90014a1f] + PDMats v0.11.0
  [d96e819e] + Parameters v0.12.2
  [e409e4f3] + PoissonRandom v0.4.0
  [f27b6e38] + Polynomials v1.2.1
  [85a6dd25] + PositiveFactorizations v0.2.4
  [21216c6a] + Preferences v1.2.2
  [92933f4c] + ProgressMeter v1.6.2
  [1fd47b50] + QuadGK v2.4.1
  [fb686558] + RandomExtensions v0.4.3
  [e6cf234a] + RandomNumbers v1.4.0
  [3cdcf5f2] + RecipesBase v1.1.1
  [731186ca] + RecursiveArrayTools v2.11.4
  [f2c3362d] + RecursiveFactorization v0.1.12
  [189a3867] + Reexport v1.0.0
  [ae029012] + Requires v1.1.3
  [79098fc4] + Rmath v0.7.0
  [7e49a35a] + RuntimeGeneratedFunctions v0.5.2
  [476501e8] + SLEEFPirates v0.6.19
  [1bc83da4] + SafeTestsets v0.0.1
  [0bca4576] + SciMLBase v1.13.2
  [efcf1570] + Setfield v0.7.0
  [699a6c99] + SimpleTraits v0.9.3
  [a2af1166] + SortingAlgorithms v1.0.0
  [276daf66] + SpecialFunctions v1.4.1
  [aedffcd0] + Static v0.2.4
  [90137ffa] + StaticArrays v1.2.0
  [82ae8749] + StatsAPI v1.0.0
  [2913bbd2] + StatsBase v0.33.8
  [4c63d2b9] + StatsFuns v0.9.8
  [7792a7ef] + StrideArraysCore v0.1.10
  [d1185830] + SymbolicUtils v0.11.2
  [0c5d862f] + Symbolics v0.1.25
  [3783bdb8] + TableTraits v1.0.1
  [bd369af6] + Tables v1.4.2
  [8290d209] + ThreadingUtilities v0.4.2
  [f269a46b] + TimeZones v1.5.5
  [a759f4b9] + TimerOutputs v0.5.9
  [a2a6695c] + TreeViews v0.3.0
  [3a884ed6] + UnPack v1.0.2
  [1986cc42] + Unitful v1.7.0
  [3d5dd08c] + VectorizationBase v0.20.7
  [700de1a5] + ZygoteRules v0.2.1
  [f5851436] + FFTW_jll v3.3.9+7
  [e33a78d0] + Hwloc_jll v2.4.1+0
  [1d5cc7b8] + IntelOpenMP_jll v2018.0.3+2
  [94ce4f54] + Libiconv_jll v1.16.1+0
  [856f044c] + MKL_jll v2021.1.1+1
  [efe28fd5] + OpenSpecFun_jll v0.5.4+0
  [f50d1b31] + Rmath_jll v0.3.0+0
  [02c8fc9c] + XML2_jll v2.9.12+0
  [0dad84c5] + ArgTools
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [8bb1440f] + DelimitedFiles
  [8ba89e20] + Distributed
  [f43a241f] + Downloads
  [9fa8497b] + Future
  [b77e0a4c] + InteractiveUtils
  [4af54fe1] + LazyArtifacts
  [b27032c2] + LibCURL
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [ca575930] + NetworkOptions
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [1a1011a3] + SharedArrays
  [6462fe0b] + Sockets
  [2f01184e] + SparseArrays
  [10745b16] + Statistics
  [4607b0f0] + SuiteSparse
  [fa267f1f] + TOML
  [a4e569a6] + Tar
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [e66e0078] + CompilerSupportLibraries_jll
  [deac9b47] + LibCURL_jll
  [29816b5a] + LibSSH2_jll
  [c8ffd9c3] + MbedTLS_jll
  [14a3606d] + MozillaCACerts_jll
  [83775a58] + Zlib_jll
  [8e850ede] + nghttp2_jll
  [3f19e933] + p7zip_jll
Precompiling project...
  24 dependencies successfully precompiled in 63 seconds (90 already precompiled)

(test_data) pkg> precompile

julia> using DataDrivenDiffEq
```

Reply:
Okay. Got the error and will commit first thing tomorrow.